xarray/namedarray/core.py

```diff
@@ -415,15 +477,15 @@ def _nonzero(self) -> tuple[Self, ...]:
     def _as_sparse(
         self,
         sparse_format: str | Default = _default,
-        fill_value=dtypes.NA,
+        fill_value: np.typing.ArrayLike | Default = _default,
     ) -> Self:
         """
         use sparse-array as backend.
         """
         import sparse
 
         # TODO: what to do if dask-backended?
-        if fill_value is dtypes.NA:
+        if fill_value is _default:
```

Comment:
dtype.NA was only used as a default value. So why not use the _default we already have?